src/utils/verifier.js

```diff
@@ -32,17 +36,18 @@ module.exports = {
   },
 
   /**
-   * Verifies that the tweet contains the given muportDID and returns the users twitter username.
+   * Verifies that the tweet contains the given 3ID and returns the users twitter username.
    * Throws an error otherwise.
    *
-   * @param     {String}            did             The muport DID of the user
+   * @param     {String}            did             The 3ID of the user (or array of equivalent dids)
    * @param     {String}            claim           A did-JWT with claim
    * @return    {Object}                            Object containing username, proof, and the verifier
    */
   verifyTwitter: async (did, claim) => {
+    const dids = typeof did === 'string' ? [did] : did
     if (!claim) return null
     const verified = await didJWT.verifyJWT(claim)
```

Comment:
3id-resolver is not registered in this class, so this will only work if you already have an open box. This is kind of annoying.